mkdocs.yml

```diff
@@ -8,15 +8,12 @@ site_author: "Frequenz Energy-as-a-Service GmbH"
 copyright: "Copyright © 2024 Frequenz Energy-as-a-Service GmbH"
 repo_name: "frequenz-client-dispatch-python"
 repo_url: "https://github.com/frequenz-floss/frequenz-client-dispatch-python"
-# TODO(cookiecutter): "main" is the GitHub repo default branch, you might want to update it
-# if the project uses a different default branch.
 edit_uri: "edit/main/docs/"
```

Comment:
Strange, I thought I updated this to use v0.x.x instead. You should probably update it (and I should update the default branch to be named like this, I will do it now).

Reply:
Renamed, please remember to update your local branch name.